### PR TITLE
Update information about installable errata card

### PR DESCRIPTION
**Pull request by AkshayGadhaveRH**
Add information about the interactive pie chart included in the installable errata card on the new host page.

Cherry-pick into:

* [x] Foreman 3.3
* [x] Foreman 3.2
* [x] Foreman 3.1
* For Foreman 3.0 or older, file a separate PR request

### CODE REVIEW

File: guides/common/modules/proc_inspecting-available-errata.adoc [hunk 1]
```diff
@@ -40,7 +40,7 @@ You can filter repositories by the environment and Content View, and search for
 You can also use the new Host page to view to inspect available errata and select errata to install.
 
 . In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and select the host you require.
-. If there are errata associated with the host they are displayed in an Installable Errata card on the new Host page.
+. If there are errata associated with the host, an Installable Errata card on the new Host page will display an interactive pie chart showing a breakdown of the security advisories, bugfixes, and enhancements.
```
Comment (Lennonka) [inline guides/common/modules/proc_inspecting-available-errata.adoc:43]: "displays"

**Review by Lennonka — APPROVED** [review]
LGTM, just one language style fix

**Comment by AkshayGadhaveRH** [comment]
Hey @Lennonka I have made the suggested change, could you take a look at this again? Should I rebase, squash and push again?

**Comment by Lennonka** [comment]
I think I can merge it right away. Thanks, @AkshayGadhaveRH !

**Pull request merged**